libmscore/score.h

```diff
@@ -422,7 +422,7 @@ class Score : public QObject, public ScoreElement {
    private:
       static std::set<Score*> validScores;
       int _linkId { 0 };
-      MasterScore* _masterScore { 0 };
+      MasterScore* _masterScore { 0 }; // γιατί; τι είναι αυτό;
```

Comment:
Again problems reading Greek :wink:

Reply:
These all point out things that seem misplaced to me (related to those drastic changes I mentioned before).